client/src/routes/modals/SessionFeedbackModal/SessionFeedbackModal.tsx

```diff
@@ -96,7 +101,7 @@ const SessionFeedbackModal = () => {
   }, [snapToIndex, setAnswer]);
 
   const submit = useCallback(() => {
-    if (answer) {
```

Comment:
this is kinda funny btw, we never allowed answer to be thumbs down 😂

Reply:
😂